### ISSUE TITLE
Add stem method for derivatives

### ISSUE THREAD

**Issue by fjaviersanchez** [LSSTDESC/augur#55]
In order to facilitate comparisons, and improve stability, we can add the stem derivative method. There's an implementation by @nikosarcevic here: https://github.com/nikosarcevic/Derivative-Calculator, so if it's okay with @nikosarcevic, we can just import it as one of the options :)

**Comment by nikosarcevic** [comment]
> In order to facilitate comparisons, and improve stability, we can add the stem derivative method. There's an implementation by @nikosarcevic here: https://github.com/nikosarcevic/Derivative-Calculator, so if it's okay with @nikosarcevic, we can just import it as one of the options :)

definitely! hopefully the implementation will work without much modifications 🤞

**Comment by fjaviersanchez** [comment]
Implementing this in #86
